src/vellum/workflows/nodes/displayable/subworkflow_deployment_node/node.py

```diff
@@ -155,14 +155,36 @@ def _compile_subworkflow_inputs_for_direct_invocation(self, workflow: "BaseWorkf
             filtered_inputs = {k: v for k, v in self.subworkflow_inputs.items() if v is not None}
             return inputs_class(**filtered_inputs)
 
+    def _get_release_id(self, deployment_name: str, release_tag: str) -> Optional[str]:
+        """Fetch the release ID for a given deployment name and release tag."""
+        try:
+            release = self._context.vellum_client.workflow_deployments.retrieve_workflow_deployment_release(
+                id=deployment_name, release_id_or_release_tag=release_tag
+            )
+            return str(release.id)
+        except Exception:
+            return None
+
     def _run_resolved_workflow(self, resolved_workflow: "BaseWorkflow") -> Iterator[BaseOutput]:
         """Execute resolved workflow directly (similar to InlineSubworkflowNode)."""
-        with execution_context(parent_context=get_parent_context()):
-            subworkflow_stream = resolved_workflow.stream(
-                inputs=self._compile_subworkflow_inputs_for_direct_invocation(resolved_workflow),
-                event_filter=all_workflow_event_filter,
-                node_output_mocks=self._context._get_all_node_output_mocks(),
-            )
+        try:
+            with execution_context(parent_context=get_parent_context()):
+                subworkflow_stream = resolved_workflow.stream(
+                    inputs=self._compile_subworkflow_inputs_for_direct_invocation(resolved_workflow),
+                    event_filter=all_workflow_event_filter,
+                    node_output_mocks=self._context._get_all_node_output_mocks(),
+                )
+        except WorkflowInitializationException as e:
```

Comment:
We should incorporate `WorkflowInitializationException` handling during the instantiation of `resolved_workflow`